packages/apps/src/SideBar/index.tsx

```diff
@@ -27,6 +30,38 @@ type Props = I18nProps & {
   toggleMenu: () => void
 };
 
+const Toggle = styled.img`
+  background: ${theme('theme', {
```

Comment:
My only comment here is that I almost feel that these values (i.e. background), should be set somewhere globally - it makes it easier to align colors as well. (That is the nice things about the current theme location - it is in a single place in terms of global colors.)

Just worried that we end up repeating some stuff spread-out all-over?

Thoughts?

Reply:
Yes, perhaps we could utilise refs to refer to abstract `styled.css` objects and inject them elsewhere - will have a look at implementations and get back to this.